ledger/catchupaccessor.go

```diff
@@ -174,6 +174,7 @@ func (c *CatchpointCatchupAccessorImpl) SetLabel(ctx context.Context, label stri
 // ResetStagingBalances resets the current staging balances, preparing for a new set of balances to be added
 func (c *CatchpointCatchupAccessorImpl) ResetStagingBalances(ctx context.Context, newCatchup bool) (err error) {
 	wdb := c.ledger.trackerDB().wdb
+	// TODO: db metrics
```

Comment:
The `CatchpointCatchupAccessorImpl` is used only during fast catchup, so we won't get any metrics on normal operations.